plugins/hemingway.js

```diff
@@ -35,14 +35,14 @@ class Hemingway extends GameBase {
 	// sent upon the creation of each game, as well as a command that can automatically retrieve the
 	// info.
 	onSignups() {
-		let announcement = `/addhtmlbox <center><b><u>Rundown of ${name}</u></b></center><ul>\
+		let announcement = `/addhtmlbox <details><summary style="text-align: center;"><b><u>Rundown of ${name}</u></b></summary>
```

Comment:
I would probably specify `Rundown of ${name} (Click to expand)` simply because some users aren't aware of the fact that you can click to toggle the collapsible. (I've seen this with posting !code blocks for suggestions/critique on writing).

Reply:
Done.